VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.html

```diff
@@ -1,15 +1,39 @@
 <ng-container *ngIf="hearingValidated; else loading">
   <h1 class="govuk-heading-xl govuk-!-margin-top-8">{{ 'quick-participant-landing-page.page-title' | translate }}</h1>
 
-  <p>{{ 'quick-participant-landing-page.paragraph1' | translate }}</p>
+  
 
   <form [formGroup]="quickLinkForm" class="govuk-form-group" (ngSubmit)="onSubmit()">
+
+    <div *ngIf="specialCharError" class="govuk-error-summary" aria-labelledby="error-summary-title" data-module="govuk-error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+        {{ 'quick-participant-landing-page.problem-with-form' | translate }}
+      </h2>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <a class="govuk-link" href="javascript:void(0)" (click)="focusToInput()">
```

Comment:
I'm a little unsure of using href="javascript:void(0)" to prevent navigation. Let me see if there's a preferred way to do this in Angular.

Reply:
cool.
I have seen this way to prevent navigation is present in many areas already.

Reply:
You can leave this as it is. Doesn't seem like there's an agreed way. [routerLink]="" is also an option.
And it seems this is an acceptable use for a tag which was another concern I had.

Reply:
You can leave this as it is. Doesn't seem like there's an agreed way. [routerLink]="" is also an option.
And it seems this is an acceptable use for a tag which was another concern I had.